docs/go-storage/services/storj.md

```diff
@@ -1,3 +1,46 @@
+[STORJ(Decentralized Cloud Storage)](https://www.storj.io)
+
 ## Project
 
-<https://github.com/beyondstorage/go-service-storj>
\ No newline at end of file
+<https://github.com/beyondstorage/go-service-storj>
+
+## Config
+
+### Storager
+
+#### Available Pairs
+
+| Name | Required | Comments |
+| ---- | -------- | -------- |
+| [credential](go-storage/pairs/credential.md) | Y | credential |
+| [work_dir](go-storage/pairs/work_dir.md) | N | work dir |
```

Comment:
Are `name` and `location` requried according to https://github.com/beyondstorage/go-service-storj/blob/master/tests/utils_test.go#L17-L22?

Reply:
Thx! I forgot to add the name pair. As you mentioned, name is required but location is not required, and I've fixed the problem.